.github/.github/pull_request_template.md

```diff
@@ -0,0 +1,54 @@
+# 📄 PR Description
+
+[Add a description for this pull request. Follow the checklist below.] 
+<!--- Describe your changes in detail including a relevant motivation and context for this change -->
+
+
+# ✒️ Checkliste PR-Author
+
+## Before submitting the PR, please make sure you do the following
+
+### Prepare Code for PR
+* [ ] The automated tests were executed locally (`mvn clean test`), and it was ensured that they passed.
+* [ ] All commented-out code and debug statements have been removed.
+* [ ] A test is included that fails without this PR but passes with it.
+* [ ] The relevant documentation, including code comments and public documentation, has been updated.
```

Comment:
```suggestion
* [ ] The relevant documentation, including code comments and public documentation, has been updated in a corresponding PR.
```

I think it does not make sense to update the productive documentation before the changes actually go live.

Reply:
Yes, I can partly understand that. But in my opinion, it definitely makes sense to update the documentation in the code. I think the public documentation should also be adapted as soon as possible. However, is it difficult to guarantee that the check will be carried out later. What do you think?
You can't guarantee at this point that the documentation will be adapted at a late stage. The idea here was to only allow changes to the production code once the documentation has also been adapted. But it true, now you might create a public documentation that is ahead of the actual code, which wouldn't be good either.

Reply:
```suggestion
* [ ] The documentation in the code has been updated and corresponds to the team guidelines.
* [ ] A person has been assigned to update the public documentation as soon as possible if this PR requires changes to it.
```

What do you think about this instead of the suggestion above?

Reply:
That's better!